velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -787,6 +787,19 @@ void windowFuzzer(
     bool orderableGroupKeys,
     const std::optional<std::string>& planPath,
     std::unique_ptr<ReferenceQueryRunner> referenceQueryRunner) {
+  auto fuzzerOptions =
+      AggregationFuzzerBase::getFuzzerOptions(timestampPrecision);
+  // The window fuzzer often runs into OOMs when generating complex types
+  // nested inside other complex types.
+  // For e.g if we were to run map_union on a MAP<MAP<INT, TIMESTAMP>, ARRAY<>>,
+  // Then currently with the worst case the inner map can have upto 10k entries
+  // as that is the limit set by complexElementsMaxSize.
+  // Window vector sizes can be 1000 rows, and each row will have the associated
+  // map_union result with another vector with inner elements of
+  // complexElementsMaxSize size. This leads to very fat vectors and an
```

Comment:
nit: Maybe rephrase this to something like "Window frames can contain tens or hundreds of rows, causing the map_union result of every input row to contain up to `windowFrameSize * complexElementsMaxSize` number of nested elements. This leads to ..."

Also, let's add one sentence at the end: "So we limit complexElementsMaxSize to 100 to avoid OOM in such a situation".